packaging/suse/portusctl/lib/configurator.rb

```diff
@@ -50,9 +50,10 @@ def ssl
         "-e", @options["ssl-email"],
         "-c", @options["ssl-country"],
         "-l", @options["ssl-city"],
-        "-s", @options["ssl-state"]
+        "-s", @options["ssl-state"],
+        "-x", @options["digest"]
       ]
-      Runner.exec("gensslcert", args)
+      Runner.exec("/srv/Portus/packaging/suse/portusctl/bin/gensslcert", args)
```

Comment:
Wow, I didn't realize we were going to vendor the genssl script. I don't like this approach because it makes things harder to maintain. If the script does not offer this possibility we should fix the script inside of the SUSE package.